fileio.c

```diff
@@ -0,0 +1,41 @@
+/* Jared Wasserman -- fileio.c */
```

Comment:
Thanks. That's odd, it works with any number of floats for me.
<img width="336" alt="screen shot 2016-08-06 at 5 30 19 pm" src="https://cloud.githubusercontent.com/assets/8939148/17459162/79848f7e-5bfb-11e6-9f57-c176c9f68935.png">